### PR TITLE
 tautulli 2.13.4 (new cask)

### PR DESCRIPTION
**Pull request by manakbisht**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/t/tautulli.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "tautulli" do
+  version "2.13.4"
+  sha256 "820842c614461ddab95f0ad3be65fdd1b2ab555074eba5a83fa3689cad2cffe1"
+
+  url "https://github.com/Tautulli/Tautulli/releases/download/v#{version}/Tautulli-macos-v#{version}-x64.pkg",
+      verified: "github.com/Tautulli/Tautulli/"
+  name "Tautulli"
+  desc "Open-source monitoring, analytics and notifications tool for Plex Media Server"
```
Comment (bevanjkay) [inline Casks/t/tautulli.rb:8]:
```suggestion
  desc "Monitoring, analytics and notifications tool for Plex Media Server"
```

File: Casks/t/tautulli.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  homepage "https://tautulli.com/"
+
+  livecheck do
+    url :url
+    strategy :github_releases
+  end
+
```
Comment (bevanjkay) [inline Casks/t/tautulli.rb:15]: You should be able to omit the `livecheck` block, as the default `:git` strategy will be inferred automatically from the `url`.

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/t/tautulli.rb (hunk 2 above)
Comment (samford) [reply Casks/t/tautulli.rb:15]: The cask uses a GitHub release asset (not a tag archive), so we should use either `GithubLatest` or `GithubReleases`. We only use `GithubReleases` when `GithubLatest` doesn't work as expected and we need to check more than one release (usually only necessary for unusual upstream situations), as it's slower and transfers more data.

Is there any reason why `GithubLatest` can't be used and `GithubReleases` is necessary? At the moment, they both return 2.13.4 as the newest version and unstable versions also look to be properly marked as pre-release (e.g., [v2.12.0-beta](https://github.com/Tautulli/Tautulli/releases/tag/v2.12.0-beta)), so `GithubLatest` seems adequate to me.

File: Casks/t/tautulli.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  url "https://github.com/Tautulli/Tautulli/releases/download/v#{version}/Tautulli-macos-v#{version}-x64.pkg",
+      verified: "github.com/Tautulli/Tautulli/"
+  name "Tautulli"
+  desc "Open-source monitoring, analytics and notifications tool for Plex Media Server"
+  homepage "https://tautulli.com/"
+
+  livecheck do
+    url :url
+    strategy :github_releases
```
Comment (samford) [inline Casks/t/tautulli.rb:13]:
```suggestion
    strategy :github_latest
```

**Comment by manakbisht** [comment]
@samford I don't think the explicit `delete` is required. The CI is failing because the install step is timing out due to a start/close dialog that is triggered by the _pkg's_ `postinstall` script.

**Comment by krehel** [comment]
There is precedent for applications which force user interactivity on install and uninstall to face rejection because of the inability to complete CI or allow users to install non-interactively.

**Comment by miccal** [comment]
Thank you for your submission @manakbisht, but we cannot accept it at this time because the `app` bundle `Tautulli.app` inside the `pkg` file `Tautulli-macos-v2.13.4-x64.pkg` does not pass our signature verification check that are a requirement for new submissions:

> macOS on ARM requires software to be signed. Please contact the upstream developer to let them know they should sign and notarize their software.

Please consider hosting this [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) and if the developers sign their app in the future we would then be happy to consider adding this to `homebrew-cask`.

**Pull request closed without merging**